### ISSUE TITLE
tour: Exported names, Pizza example was confusing

### ISSUE THREAD

**Issue by bhavaniravi** [golang/tour#1220]
Context: https://tour.golang.org/basics/3

The `Pizza` example was out of the blue for someone who has never seen go before. I was wondering if it was a part of math, but why would it be. There was no example on the right side with a custom package.

Issues #592 and #683 Also comes from the same place.

It would be better if 
1. Explicitly mention Pizza is just an example
2. Add another exported name from `math` or different package

**Comment by TommyMurphyTM1234** [comment]
I totally agree. I too was completely confused about the `Pizza` non sequitur. The fact that there have been three separate issues logged about this is evidence that there's a genuine problem here.
